### DISCUSSION TITLE
What is CSSMode?

### DISCUSSION THREAD

**Discussion by HarmonyEarth** [nolimits4web/swiper#6513] [Q&A]
What does CSS mode do? There's no explanation in the documentation.

**Comment by Arhell** [comment]
https://swiperjs.com/swiper-api#param-cssMode
https://swiperjs.com/types/interfaces/types_swiper_options.swiperoptions#cssmode

**Reply by HarmonyEarth** [reply]
Thank you, I don't know how I missed this. Hopefully the team will add search functionality to the documentation.

**Reply by Arhell** [reply]
Used a browser search and a little Google

**Comment by HarmonyEarth** [comment]
Thank you for the help!

**Reply by Arhell** [reply]
You are welcome!
